partner_phone_search/models/res_partner.py

```diff
@@ -15,14 +15,15 @@ def name_search(self, name, args=None, operator='ilike', limit=100):
             domain = ['|', '|', ('phone', operator, name),
                       ('mobile', operator, name), ('email', operator, name)
                       ]
-            partners = self.search(domain + args, limit=limit,)
+            partners = self.search(domain + args, limit=limit)
             res = partners.name_get()
             if limit:
                 limit_rest = limit - len(partners)
             else:
                 limit_rest = limit
             if limit_rest or not limit:
                 args += [('id', 'not in', partners.ids)]
-                res += super(ResPartner, self).name_search(
-                    name, args=args, operator=operator, limit=limit_rest)
+                res += super().name_search(name, args, operator, limit_rest)
```

Comment:
You should call keyword arguments with keywords. So limit your change to removing the unneeded arguments from the super() call.